src/modules/image-upload/ImageUploadHelper.js

```diff
@@ -138,63 +141,31 @@ export default class ImageUploadHelper {
 	}
 
 	async send(name: string, file: File | { uri: string; type: string; }): Promise<UploadResult> {
```

Comment:
I’d have thought the signature should be either `async send(...): UploadResult` or `send(...): Promise<UploadResult>`. I guess flow doesn't fully understand async?